backend/app/services/channels/handler.py

```
@@ -725,6 +764,414 @@ async def _handle_command(
+            task = im_task_continuation_service.validate_personal_wework_task(
+                db, user.id, task_id
+            )
+        except Exception:
+            self.logger.exception(
+                "[%sHandler] Failed to bind private IM task: user_id=%s task_id=%s",
+                self._channel_type.value,
+                user.id,
+                task_id,
+            )
+            await self.send_text_reply(
+                message_context, "未找到可绑定的个人任务，请使用 /switch 重新选择。"
```

Ruff (0.15.17)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)